exporter/debugexporter/README.md

```diff
@@ -36,6 +36,8 @@ The following settings are optional:
   Refer to [Zap docs](https://godoc.org/go.uber.org/zap/zapcore#NewSampler) for more details
   on how sampling parameters impact number of messages.
 - `use_internal_logger` (default = `true`): uses the collector's internal logger for output. See [below](#using-the-collectors-internal-logger) for description.
+- `output_paths`: (optional, only used if `use_internal_logger: false`) list of file paths or URLs to write logging output to. Defaults to `["stdout"]`.
```

Comment:
Let's not mention "URLs", it may be confusing to users. I propose to officially support only bare file paths, regardless of what is [internally supported by Zap](https://pkg.go.dev/go.uber.org/zap@v1.27.0#Open).

Let's also mention the only special strings: "stdout" and "stderr".
```suggestion
- `output_paths`: (optional, only used if `use_internal_logger: false`) list of file paths to write logging output to. Defaults to `["stdout"]`. Special strings are "stdout" and "stderr".
```